Data/ByteString/Lazy/Char8.hs

```diff
@@ -196,7 +198,7 @@ module Data.ByteString.Lazy.Char8 (
   ) where
 
 -- Functions transparently exported
-import Data.ByteString.Lazy 
+import Data.ByteString.Lazy
```

Comment:
Best to not mix unrelated whitespace changes with a feature change.  Any whitespace cleanup not immediately adjacent to code changed as part of the feature should be a separate commit.

Reply:
Ya, fair point. I'll clean that up!

Reply:
I'd actually be happy to include the whitespace changes now that they've already been committed. I think this PR is simple enough that they don't hurt too much.

Reply:
+1 to @sjakobi 
Adding trailing spaces is a bad thing, but cleaning them is totally acceptable.